modules/leaflet/test/index.spec.ts

```diff
@@ -0,0 +1,12 @@
+// deck.gl-community
```

Comment:
Fine, though not super excited about files called `index...` maybe we could name it after the tests inside?